package.json

```diff
@@ -33,23 +33,21 @@
     "istanbul": "~0.3.5",
     "map-stream": "~0.1.0",
     "mocha": "~2.3.3",
-    "nock": "^2.13.0"
+    "nock": "^2.13.0",
+    "tslint": "^5.11.0",
+    "tslint-eslint-rules": "^5.4.0",
+    "typescript": "^3.1.3"
   },
   "scripts": {
-    "compile": "babel --optional runtime -d distrib/npm/ src/",
+    "compile": "tsc",
+    "compile-babel": "babel --optional runtime -d distrib/npm/ src/",
```

Comment:
why two compiles? Can we get rid of Babel now that we're using TS?

Reply:
Good point - yes, I think we can get rid of Babel